### PR TITLE
Add Kotlin migration article

### PR DESCRIPTION
**Pull request by amahdy**

### CODE REVIEW

File: tutorials/jvm/migrate-to-kotlin/content.adoc [hunk 1]
```diff
@@ -0,0 +1,127 @@
+= Migrate Java Enterprise Apps to Kotlin
+
+:type: text, image
+:topic: jvm
```
Comment (paulroemer) [inline tutorials/jvm/migrate-to-kotlin/content.adoc]: The topic is already extracted from the folder strucutre

Comment (paulroemer) [reply tutorials/jvm/migrate-to-kotlin/content.adoc]: And the topic 'jvm' does not make much sense. Why not just 'migration'?

Comment (amahdy) [reply tutorials/jvm/migrate-to-kotlin/content.adoc]: It's JVM languages, but migration also make sense. We will have a lot of migration tutorials.

So should I remove the `:topic:` line? I have it in pretty much all `content` files, copied from other files as well.

Comment (paulroemer) [reply tutorials/jvm/migrate-to-kotlin/content.adoc]: @marcushellberg?

Comment (marcushellberg) [reply tutorials/jvm/migrate-to-kotlin/content.adoc]: Topic should only come from the folder structure. I may still have some incorrectly in my older articles. https://raw.githubusercontent.com/vaadin-learning-center/learning-content/development/TEMPLATE.adoc should be up to date with what we have agreed

**Comment by marcushellberg** [comment]
Review notes:
- Please don't use any fancy quotes, just plain " and '
- The source code for the finished application should preferably be under the vaadin-learning-center organization. 
- Source code blocks use `----` not ```

**Comment by amahdy** [comment]
@Haprog it seems that there are some issues here from your recent merge 524da44
Can you please merge this one when everything else is clear? Thanks!

**Comment by Haprog** [comment]
@amahdy That other merge you mentioned didn't have anything to do with this one, but I merged this manually and updated it to use the new directory structure so that it works. Also fixed few other minor issues I noticed in the content formatting.

**Comment by amahdy** [comment]
Thanks @Haprog !

**Pull request merged**